docs/DeveloperGuide.md

```diff
@@ -118,13 +119,14 @@ The `UI` component,
 
 **API** : [`Logic.java`](../src/main/java/seedu/address/logic/Logic.java)
 
-1. `Logic` uses the `Parser` class to parse the user command.
-2. This results in a `Command` object which is executed by the `LogicManager`.
-3. The command execution can affect the `Model` (e.g. adding a person) and/or raise events.
-4. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
+1. `Logic` invokes the `Parser` class to parse the user command.
+2. Upon recognizing the command, it sends the arguments to the `CommandParser` class, which recognizes the remaining arguments.
+3. This information is then captured in a `Command` object which is executed by the `LogicManager`.
+4. The command execution can affect the `Model` (e.g. adding a person) and/or raise events.
+5. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
 
 Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete 1")`
- API call.<br>
+ API call.  
 <img src="images/DeletePersonSdForLogic.png" width="800"><br>
 
```

Comment:
Good job updating the class diagrams according to your new product. I think you can rename the image as well.
But here I see DeletePersonSdForLogic is about deleting a person. Are you really having such a use case. If not do necessary changes to reflect whatz in your system